Prace_domowe/Praca_domowa2/Mateusz_Ziemła/pd2.ipynb

```diff
@@ -0,0 +1,737 @@
+{
```

Comment:
<p>Fajnie jakbyś to skomentował, bo na pierwszy rzut oka nie widać czym się&nbsp;różnią te dwie pętle</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/60/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>